client/lib/post-normalizer/rule-wait-for-images-to-load.js

```diff
@@ -58,6 +58,15 @@ export default function waitForImagesToLoad( post ) {
 				return find( post.images, { src: image.src } );
 			} ), Boolean );
 
+			// this adds naturalHeight and naturalWidth to the images
```

Comment:
might need to rebase and see if this is still true after https://github.com/Automattic/wp-calypso/pull/9238